package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@babel/types": "^7.24.0",
     "@eslint-types/typescript-eslint": "^7.2.0",
+    "@playwright/test": "^1.42.1",
```

Comment:
We can remove this one now, no?

Reply:
This should be a separate discussion too, but I think we could consider using `expect` from `@playwright/test` where it's appropriate since it has some neat feature to write robust assertions.

For now, I'll revert this back too. Let me check.